### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by swethaogeti**
created homepage: navbar,banner,countDown, catagoryCards,footer

### CODE REVIEW

**Review by BISINIGISHIVANI — APPROVED** [review]
nice ui of homepage @swethaogeti

**Review by MaruthiPothuganti — COMMENTED** [review]
Take care of Code Indentation in some of the files. Apart from that, Everything else looks good to go. User Interface is minimalistic and looking great.

**Comment by swethaogeti** [comment]
> Take care of Code Indentation in some of the files. Apart from that, Everything else looks good to go. User Interface is minimalistic and looking great.

done!! fixed indentation as suggested.

File: src/components/slider/slider.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+@media (max-width:1200px){
+    html{
+        font-size: 55%;
+    }
+}
+
+@media (max-width:991px){
+
+}
```
Comment (kotesh-arya) [inline src/components/slider/slider.css:62]: You might have overlooked this media query, check this out

File: src/components/catagories/CatagoryCard.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { catagoryData } from '../../utils/data'
+import './catagoryCard.css'
+console.log(catagoryData)
```
Comment (kotesh-arya) [inline src/components/catagories/CatagoryCard.js:4]: Try to follow the habit of clearing Console.log's before merging 👍🏻

**Review by kotesh-arya — APPROVED** [review]
Suggested Minor improvements, do check them. Rest of the code looks good enough for a merge.

**Pull request closed without merging**